### DISCUSSION TITLE
This project dead / abandoned?

### DISCUSSION THREAD

**Discussion by XtremeOwnageDotCom** [atifaziz/NCrontab#103] [Q&A] [answered]
Was looking for a good library for parsing cron schedules, came across this one.

But, a bit weary as there are untouched issues since 2020. The last commit was over a year ago....

And... well, there doesn't seem to be much activity here.

**Comment by atifaziz** [comment, marked as answer]
See discussion #101.

**Reply by XtremeOwnageDotCom** [reply]
Excellent, thanks for the response.
